core/src/apps/common/passphrase.py

```diff
@@ -38,8 +39,11 @@ async def _request_on_host() -> str:
     )
     from trezor.wire.context import call
 
-    request_passphrase_on_host()
+    async def _delay_request_passphrase_on_host() -> None:
+        await loop.sleep(100)
+        return request_passphrase_on_host()
 
+    workflow.spawn(_delay_request_passphrase_on_host())
     request = PassphraseRequest()
```

Comment:
this seems to rely on the spawned task being silently killed in the background by:
1. displaying the passphrase keyboard, or
2. displaying the "next screen will show passphrase", or
3. displaying any UI element in the following workflow, or
4. session reset at the end of the spawning flow

(1) to (3) rely on `workflow.close_others` being called by `interact`, which may not be the case in the future (though maybe it will? there's a comment in Layout.start saying that it's a good idea to do it, perhaps _exactly_ to avoid this sort of situation)

(4) will not be the case, esp. if we build a proper mechanism that tracks running workflows before clearing the session

the spawned task should be explicitly `close()`d after receiving the PassphraseAck